Buildfile

```diff
@@ -7,6 +7,6 @@ cookbook.depends 'propsd' do |propsd|
   propsd.path './cookbook'
 end
 
-profile :default do |default|
-  default.chef.run_list 'propsd::default'
+profile :default do |test|
```

Comment:
I still don't love that `default` maps to `test` but you're right that at this point there's no distinction.